File: week1/d3/algo-deaf-grandma-master/deaf_grandma.js
function deafGrandma() {
    let goodbyeCount = 1;
    while (true) {
        let response = prompt("What to say to grandma. ")
        if (!response) {
            console.log("WHAT?!")
        }
        else if (response[0] === response[0].toLowerCase()) {
            console.log("SPEAK UP KID!")
        }
        else if (response === "GOODBYE!" && goodbyeCount < 1) {
            console.log("LEAVING SO SOON?")
            goodbyeCount++;
        }
        else if (response === "GOODBYE!" && goodbyeCount === 1) {
            console.log("LATER, SKATER!")
            break;
        }
        else if (response[0] === response[0].toUpperCase()) {
            console.log("NO, NOT SINCE 1946!")
        }   
    }
}

deafGrandma();

File: week1/d3/algo-deaf-grandma-master/deaf_grandma.py
def deaf_grandma():
    goodbye_counter = 0
    while True:
        response = input("What to say to grandma? ")
        if not response:
            print("WHAT!")
        elif response[0] == response[0].lower():
            print("SPEAK UP, KID!")
        elif response == "GOODBYE!" and goodbye_counter < 1:
            print("LEAVING SO SOON?")
            goodbye_counter += 1
        elif goodbye_counter == 1:
            print("LATER, SKATER!")
            break
        else:
            print("NO, NOT SINCE 1946!")

deaf_grandma()
